doc/neps/nep-0018-array-function-protocol.rst

```diff
@@ -84,27 +87,28 @@ We propose the following signature for implementations of
 
 .. code-block:: python
 
-    def __array_function__(self, func, types, args, kwargs)
+    def __array_function__(self, func, possibly_overloaded, args, kwargs)
 
 -  ``func`` is an arbitrary callable exposed by NumPy's public API,
    which was called in the form ``func(*args, **kwargs)``.
--  ``types`` is a list of types for all arguments to the original NumPy
-   function call that will be checked for an ``__array_function__``
-   implementation.
--  The tuple ``args`` and dict ``**kwargs`` are directly passed on from the
+-  ``types`` is a list of argument types from the original NumPy
```

Comment:
The name now has changed from `types` to `possibly_overloaded` - should add that I think this is a terrible name! Just `overloaded` would be better.

(Separate: still think an `OrderedDict` would be better - but have to look at implementation.)